sihl/src/contract_user.ml

```diff
@@ -34,10 +34,29 @@ module type Sig = sig
     val user_from_token
       :  ?key:string
       -> (string -> k:string -> string option Lwt.t)
-      -> Rock.Request.t
+      -> Opium.Request.t
       -> t option Lwt.t
 
-    (** [user_from_session ?cookie_key ?secret ?key ?secret request] returns the
+    (** [user_to_token ?key create_token user response] returns a response with
+        a [Bearer] token set to the [Authorization] header that has the [user]
+        associated to it. This is typically part of the login workflow. Any
+        previously set [Authorization] header is overwritten. This is typically
+        part of the login workflow.
+
+        [create_token] is a function that associates data to a string.
+        [sihl-token] can be used here. *)
```

Comment:
Do you want to point to a specific function from this package?

Reply:
No, `sihl` does not know `sihl-token`.